src/components/main/person/PersonDetail.vue

```diff
@@ -0,0 +1,165 @@
+<script>
+  import personService from 'services/person';
+  import { Loading } from 'element-ui';
+
+  export default {
+    components: { Loading },
+
+    data() {
+      return {
+        person: {},
+        isLoading: false,
+        loadingInstance: {}
+      };
+    },
+
+    mounted() {
+      this.isLoading = true;
+      this.loadingInstance = Loading.service({ fullscreen: true });
+      this.getPerson(this.$route.params.personId);
+    },
+    route: {
+      data: ({ to: { params: { personId } } } ) => ({
+        person: personService.get(personId)
+      })
+    },
+    methods: {
+      getPerson(personId) {
+        personService.getById(personId)
+          .then((person) => {
+            this.person = person;
+            this.isLoading = false;
+            this.loadingInstance.close();
+            console.log(this.person.name);
+          });
+      }
+    }
+  };
+</script>
+
+<template lang='pug'>
+  <el-row :gutter="20" align="middle">
```

Comment:
Lets use `pug`  instead of html 🙂, here is a good resource to translate it http://html2jade.org/